core/src/templates/components/card/card.twig

```diff
@@ -85,7 +85,7 @@
 
     {# Big ol button. Adds a BEM class for targeting. #}
     {%- if card_button_label is not empty and allow_links %}
-      <div class="su-card__button">
+      <button class="su-card__button">
```

Comment:
@cjwest This cannot be a `<button>` as it contains an `<a>` tag.

Reply:
Hah!

> Be careful when marking up links with the button role. Buttons are expected to be triggered using the Space key, while links are expected to be triggered through the Enter key.